packages/amplify-e2e-tests/src/setup-tests.ts

```diff
@@ -18,3 +18,13 @@ jest.setTimeout(JEST_TIMEOUT);
 if (process.env.CIRCLECI) {
   jest.retryTimes(1);
 }
+
+beforeEach(async () => {
```

Comment:
There are more `setup-tests.ts` files in the repo. Please make similar change there too (duplication is fine, alternatives are costly and fragile).